index.html

```diff
@@ -8,9 +8,10 @@
   </head>
   <body>
     <div id="app">
-      <div class="cart-count"></div>
+      <div class="cart-count" onclick="changeClass('cart')"></div>
```

Comment:
Avoid adding JavaScript to your HTML elements. Also, I don't understand what this is doing for you.